core/backend/src/test/standalone/ChangeMerge.test.ts

```diff
@@ -24,7 +24,7 @@ import { HubMock } from "../../HubMock";
 import { RebaseChangesetConflictArgs, TxnArgs } from "../../internal/ChangesetConflictArgs";
 import { IModelTestUtils, TestUserType } from "../IModelTestUtils";
 chai.use(chaiAsPromised);
-import sinon = require("sinon"); // eslint-disable-line @typescript-eslint/no-require-imports
+import * as sinon from "sinon";
```

Comment:
I would import the individual APIS from sinon, rather than import *

Reply:
updated